leda/components/Collapse/CollapseBody.tsx

```diff
@@ -74,6 +75,7 @@ export const Body = React.forwardRef((props: BodyProps, ref?: React.Ref<HTMLElem
 
   return (
     <BodyWrapper
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
You should not use @ts-ignore, please use @ts-expect-error with proper description

Reply:
Proper description needs proper investigation, it is not a priority now